oak-search-elastic/src/main/java/org/apache/jackrabbit/oak/plugins/index/elastic/index/ElasticIndexWriter.java

```diff
@@ -186,11 +185,7 @@ private void provisionIndex() throws IOException {
         }
 
         final CreateIndexRequest request = ElasticIndexHelper.createIndexRequest(indexName, indexDefinition);
-        if (LOG.isDebugEnabled()) {
-            StringBuilder sb = new StringBuilder();
-            JsonpUtils.toString(request, sb);
-            LOG.debug("Creating Index with request {}", sb);
-        }
+        LOG.debug("Creating Index with request {}", request);
```

Comment:
I think this will not work properly... I think CreateIndexRequest doesn't have a toString() method